docs/tasks/capi-operator/add_infrastructure_provider.md

```diff
@@ -1,7 +1,48 @@
 ---
-sidebar_position: 1
+sidebar_position: 2
 ---
 
 # Add Infrastructure Provider
 
-In this section we will cover how to add an infrastructure proviedr such as AWS.
\ No newline at end of file
+This section describes how an infrastructure provider such as `Azure` could be added using `Cluster API Operator`.
+
+### Installing Azure Infrastructure Provider
```

Comment:
We can always pin it to a version of the operator docs to use, and the docs are describing a fixed state of some release of operator and turtles IMO. Additionally the value of the documentation is in simple access to the most common use cases in a single unified location. We may explore some better documentation options for the CAPI Operator, as a single README.md file is quite inconvenient to always refer to. It could clearly be separated into multiple sections.

Reply:
+1 for deduplicating